Include MultiOp RPC include files in the public interface.

From: Aaron Jones <aaron@vexing.codes>

Include MultiRead.h, MultiWrite.h, and MultiRemove.h in the installed includes
so the asynchronous interfaces of these operations can be employed by client
programs.
---
 GNUmakefile |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/GNUmakefile b/GNUmakefile
index 7e9dedd6..26d54640 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -458,6 +458,11 @@ INSTALL_INCLUDES := \
     src/LogMetadata.h \
     src/MasterClient.h \
     src/Minimal.h \
+    src/MultiIncrement.h \
+    src/MultiOp.h \
+    src/MultiRead.h \
+    src/MultiRemove.h \
+    src/MultiWrite.h \
     src/Object.h \
     src/ObjectBuffer.h \
     src/ObjectRpcWrapper.h \
